lib/session.js

```diff
@@ -10,7 +10,7 @@ var events = require('events'),
 ///////////////////////////////////////////////////////////
 // exports
 
-exports.create = function(debuggerPort, config) {
+exports.create = function(debuggerPort, debuggerHost, config) {
```

Comment:
I think I need to change this `debuggerHost` here to move into the last for the backward compatibility, right ? 
```
 exports.create = function(debuggerPort, config, debuggerHost) {
```